data_engineering_pulumi_components/pipelines/land_to_history.py

```diff
@@ -27,3 +30,10 @@ def __init__(
             tags=None,
             opts=ResourceOptions(parent=self),
         )
+
+        self._raw_history_bucket = RawHistoryBucket(
+            resource_name=f"{name}",
+            name=f"{name}",
```

Comment:
These no longer need to be f-strings:
```suggestion
            resource_name=name,
            name=name,
```